src/__tests__/medications/ViewMedication.test.tsx

```diff
@@ -116,11 +116,11 @@ describe('View Medication', () => {
   })
 
   it('should not display the canceled date if the medication is not canceled', async () => {
-    const { container } = setup({} as Medication, [Permissions.ViewMedication])
+    setup({} as Medication, [Permissions.ViewMedication])
 
-    await waitFor(() => {
-      expect(container.querySelector('.canceled-on')).not.toBeInTheDocument()
-    })
+    expect(
+      screen.queryByRole('heading', { name: /medications\.medication\.canceledOn/i }),
+    ).not.toBeInTheDocument()
```

Comment:
I'm a little worried there may be a spinner or something here making this test not actually doing what you may think.

If there isn't a spinner, then ignore me :P

Reply:
Nope, doesn't appear to be a spinner in this one!